### ISSUE TITLE
Error 7000 but no autosubmit.lock file or experiment running

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1111]
In GitLab by @ drodrig1 on Aug 11, 2023, 09:46

Hello @dbeltrankyl and @kinow,

### Autosubmit Version

I'm using v.3.14.0

### Expid affected(If applicable)

a69a

### Which task has issues? Where is the log(If applicable)

`autosubmit create a69a -cw `

### Summary

I was trying tests to use wrappers with autosubmit, so I was creating and killing the experiment several times. In one of these, I could not create the experiment again due to error 7000

```
(...)
Checking scripts...
270 of 1080 checked
540 of 1080 checked
810 of 1080 checked
1080 of 1080 checked
 [CRITICAL] BU [eCode=7000]
More info at https://autosubmit.readthedocs.io/en/v3.14.0/troubleshooting/error-codes.html 
```
I checked that 
`7000 Experiment is locked due another instance of Autosubmit using it Halt other experiment instances //Delete <expid>/tmp/autosubmit.lock`

But this file does not exist, neither the experiment is running `ps -ef | grep a69a`

### Steps to reproduce

Just try to create the experiment. You can see ~/tmp/ and LOGS at usual in `/esarchive/autosubmit/a69a`

`autosubmit create a69a -cw `

### What is the current bug behavior?

The experiment is not created

### What is the expected correct behavior?

The experiment is created

### Any other relevant information(if applicable)

I checked these issues and documentation but none worked for my case:
https://earth.bsc.es/gitlab/es/autosubmit/-/issues/956
https://earth.bsc.es/gitlab/es/autosubmit/-/wikis/Possible-Operational-Problems-and-Solutions
https://earth.bsc.es/gitlab/es/autosubmit/-/issues/899

()

Any ideas? Thanks!

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Aug 11, 2023, 10:13

Hello @ drodrig1 ,

Did you try to relaunch the command again? The file doesn't exist because it is deleted when the experiment is not longer locked

**Comment by kinow** [comment]
In GitLab by @ drodrig1 on Aug 11, 2023, 10:15

Yes yes, I did try this several times, and with several workarounds (other autosubmit machine, ...)

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Aug 11, 2023, 10:15

Could you try to use the 3.15.0b version?

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Aug 11, 2023, 10:18

It seems like a bug in which the wrapper interprets HERMES_BU as two different wrappers ( Hermes + BU ) one solution for 3.14 would be to change the job name ( to don't have "_"). But not sure if this was fixed in 3.15 already

**Comment by kinow** [comment]
In GitLab by @ drodrig1 on Aug 11, 2023, 10:23

I could try, but it was already working fine with vertical wrappers on the same job and in there the "_" did not show any issue. This began with the horizontal-vertical wrapper.
I could try with 3.15, I'm loading it as `autosubmit/3.15.0b-foss-2015a-Python-2.7.9`

**Comment by kinow** [comment]
In GitLab by @ drodrig1 on Aug 11, 2023, 10:26

I got the same

```
drodrig1@ bscesautosubmit01:   /esarchive/autosubmit/a69a$ autosubmit create a69a -cw -v
The a69a experiment 3.15.0b version is being updated to 3.15.0b0 for match autosubmit version
Autosubmit is running with 3.15.0b0
Preparing .lock file to avoid multiple instances with same expid.
(...)
Checking scripts...
270 of 1080 checked
540 of 1080 checked
810 of 1080 checked
1080 of 1080 checked
 [CRITICAL] BU [eCode=7000]
More info at https://autosubmit.readthedocs.io/en/v3.15.0/faq.html

```

**Comment by kinow** [comment]
In GitLab by @ drodrig1 on Aug 11, 2023, 10:31

Ok @dbeltrankyl  I modified the job name, excluding the "_" and it does work! Not sure why this happens with "horizontal-vertical" wrapper and not with only vertical or only horizontal (I tried both before). But well, if this happens again we know why already. 

Many thanks! If you want to comment something else you can comment and close.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Aug 11, 2023, 10:35

Hello @ drodrig1 ,

Good that it works! I'll remain this open to be fixed in 3.15,
